packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -55,48 +55,74 @@ export default class Meetings extends StatelessSparkPlugin {
    * In current nomenclature, this would create a `call`.
    * `spaceID` is used in {https://github.com/webex/spark-js-sdk/blob/master/packages/node_modules/@ciscospark/plugin-phone/src/call.js#L1007 `call.dial`}
    * @param {string} destination - {https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/voice/cube/configuration/cube-book/sip-supp-uri-based-dialing.html sipURL} or spaceID, AKA {https://developer.webex.com/endpoint-rooms-roomId-get.html roomID}
-   * @returns {Meeting} A new Meeting.
+   * @returns {Promise} A new Meeting.
```

Comment:
I'm fine changing these. I think I originally put what type (class) it returned but that doesn't make sense